bindgen/callbacks.rs

```diff
@@ -135,6 +135,15 @@ pub trait ParseCallbacks: fmt::Debug {
     fn process_comment(&self, _comment: &str) -> Option<String> {
         None
     }
+
+    /// Potentially override the visibility of a composite type field
+    fn field_visibility(
+        &self,
+        _type_name: &str,
```

Comment:
Could you put these arguments in a non-exhaustive struct so we can easily extend them in the future without making it a breaking change?